.github/workflows/release-assets.yml

```
@@ -18,9 +18,109 @@ on:
+        run: |
+          set -euo pipefail
+          
+          release_version="$(<<<"$BRANCH_NAME" sed 's/.*\///')"
+          if [[ -z "$release_version" ]]; then
+            echo "::error title=invalid branch name::$BRANCH_NAME isn't \"*/<version>\""
+            exit 1
+          fi
+          set -x
+          echo "version_tag=v$release_version" >> "$GITHUB_OUTPUT"
+
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)